presto-native-execution/presto_cpp/main/connectors/arrow_flight/ArrowFlightConfig.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/connectors/arrow_flight/ArrowFlightConfig.h"
```

Comment:
nit: new line above.

Reply:
I don't think we need an extra `arrow_flight` directory. All the files have ArrowFlight prefix anyway.

Reply:
There are a quite a few files for the arrow flight connector, not all of them have the prefix. For example the "auth" stuff is only for the Arrow connector, Utils for Arrow testing, and tls certs used for testing. I think it makes sense to keep all this in a directory otherwise it will be a mess for any other connectors added.